modules/cli/pages/index.adoc

```diff
@@ -506,27 +506,48 @@ $ skupper gateway generate-bundle <config-filename> <destination-directory>
 
 where:
 
-* <config-filename> - the name of the YAML file, include suffix, that you generated in the previous step.
+* <config-filename> - the name of the YAML file, including suffix, that you generated in the previous step.
 * <destination-directory> - the location where you want to save the resulting gateway bundle, for example `~/gateways`.
 
---
+For example:
+----
+$ skupper gateway generate-bundle database.yaml ./
+----
 
+This bundle contains the gateway definition YAML and a  certificate that allow access to the {service-network}.
 
+--
 
 . Copy the gateway definition file, for example, `mylaptop-jdoe.tar.gz` to the machine that hosts the service you want to expose on the {service-network}.
 
 . From the machine that hosts the service you want to expose:
 +
+--
 ----
 $ mkdir gateway
-$ tar -xvf <gateway-definition-file>
+
+$ tar -xvf <gateway-definition-file> --directory gateway
 $ cd gateway
 $ sh ./launch.py
-$ systemctl --user status <gateway-definition-name>
 ----
+
+NOTE: Use `./launch.py -t podman` or `./launch.py -t docker` to run the {skupper-router} in a container.
+
+Running the gateway bundle uses the gateway definition YAML and a certificate to access and expose the service on the {service-network}.
+
+--
+
+. Check the status of the gateway service:
 +
+--
+----
+$ systemctl --user status <gateway-definition-name>
```

Comment:
This is to check the gateway that is of type service. For the container types, the equivalent would be:

$ podman inspect <gateway-definition-name>
or 
$ docker inspect <gateway-definition-name>